src/quo2/components/avatars/user_avatar.cljs

```diff
@@ -141,7 +141,7 @@
      (if profile-picture
        ;; display image
        [fast-image/fast-image
-        {:source {:uri profile-picture}
+        {:source profile-picture
```

Comment:
hmm, you're changing the api here. Somewhere else in the code base it's probably expecting the old format? Can you either add an `or` case here or update other parts of the code base which are using user_avatar?

Reply:
e.g `if (:uri profile-picture) (:uri profile-picture) profile-picture)`, perhaps there's a cleaner solution but you're adding a change which might be breaking elsewhere in the codebase

Reply:
I have the same toughts than you about this but someone told me to change it like I did, since it's not breaking this, it's fixing this.

Reply:
thank @mattgle, did you also check in the codebase to make sure no where else is now using this incorrectly?

Reply:
@J-Son89 Would you speak to me over Discord to solve some issues with the comments? `mattgle.eth#1997`

Reply:
I think it should work now!

Reply:
> e.g `if (:uri profile-picture) (:uri profile-picture) profile-picture)`, perhaps there's a cleaner solution but you're adding a change which might be breaking elsewhere in the codebase

Hi, this will not work, its `{:uri` not `(:uri`

> I have the same toughts than you about this but someone told me to change it like I did, since it's not breaking this, it's fixing this.

Yes, it is fixing user-avatar component preview screen, but it still might be breaking other places where user-avatar is used. Better to search the codebase for uses of user-avatar and also ping the QA team while testing to notify them.